kolibri/core/templates/kolibri/base.html

```diff
@@ -7,6 +7,7 @@
   {% base_frontend_async %}
   {% base_frontend_sync %}
   <link type="text/css" rel="stylesheet" href="{% static 'core/css/components.css' %}" />
+  <link type="text/css" rel="stylesheet" href="{% static 'core/css/kolibri_style.css' %}" />
```

Comment:
Thanks for the tip @indirectlylit. @DXCanas can you address this in a separate PR?